src/ServiceControl.UnitTests/API/APIApprovals.cs

```diff
@@ -76,5 +76,16 @@ public void RootPathValue()
             Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
             Approver.Verify(JToken.Parse(result.Body.AsString()).ToString(Formatting.Indented));
         }
+
+        [Test]
+        public void TransportNames()
+        {
+            //HINT: Those names are used in PowerShell scripts thus constitute a public api
```

Comment:
We match on more than just these names though. Don't those other names also constitutes a part of the public API? If I had an existing PowerShell script that uses the old name, it will work. But if we break that, this test won't show it.

Reply:
The goal of adding the test was to warn us when we make changes to transport names that are used in PS scripts. So this tests will fail if we change the name and break existing client scripts.

Reply:
Isn't it possible that there are existing scripts that could work today that are not covered by this test?

Reply:
Hmm. I think I'm missing something from your comments @mikeminutillo. Do you have any concrete example in mind that would help me understand this better?

Reply:
If I have a script today that specifies transport as `RabbitMQ` it will work. If I update to this version, it won't. Is that correct? Isn't this a breaking change?

Reply:
It doesn't because we validate the input to the Powershell cmdlet: https://github.com/Particular/ServiceControl/blob/7efe6bc59712afb35bbfd67fc52516c25484f35c/src/ServiceControlInstaller.PowerShell/Cmdlets/ServiceControlInstances/NewServiceControlInstance.cs#L67

Reply:
In which case all the applied changes to transport names are a breaking change.

Reply:
Frustrating but correct.

Reply:
But SC doesn't have to follow semver right? So we don't have to bump major unless we want to "alert" the customers about large changes? (I don't think this would qualify)

Reply:
Powershell allows having Validation functions. So instead of having a fixed set you can execute a function that validates against something that has the matches logic. Would that be an option?

Reply:
@mikeminutillo @danielmarbach 
We have already broken PS scripts in the past in this way e.g. when adding [topologies support in ASB](https://github.com/Particular/ServiceControl/commit/1de30ac9953847b40e281c911fefcabadcfb1d10#diff-beba97d3af48daefef3bc0111c53fafeR64) - this is just an example I'm not picking this for any purpose. 

I'm not saying that this is the desired behavior long term but the assumption behind this PR was that we can make such modifications in minors.